### PR TITLE
format code

### PR DESCRIPTION
**Pull request by appt2**
hi me formating all xml and java and adding lastversion androidx and material

### CODE REVIEW

**Comment by nethical6** [comment]
Damn I really appreciate it. I was looking for someone to help me do it . Thanks a lot....

**Comment by appt2** [comment]
> Damn I really appreciate it. I was looking for someone to help me do it . Thanks a lot....

Please, I saw that your codes are messed up, I arranged them, and you were using old libraries, so this option is not recommended, at least gr is recommended version 8.2.2

File: app/build.gradle [hunk 1]
```diff
@@ -40,14 +40,11 @@ android {
...
     implementation "androidx.preference:preference:1.2.1"
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-	implementation 'com.google.android.material:material:1.4.0'
-	implementation 'com.google.code.gson:gson:2.8.8'
-    implementation 'com.squareup.okhttp3:okhttp:4.12.0'
-
+    implementation 'androidx.appcompat:appcompat:1.7.0-alpha03'
+	  implementation 'com.google.android.material:material:1.12.0-alpha03'
+  	implementation 'com.google.code.gson:gson:2.10.1'
+    implementation 'com.squareup.okhttp3:okhttp:5.0.0-alpha.12'
```
Comment (aikrq) [inline app/build.gradle:47]: where proper formatting?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,98 +1,99 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (aikrq) [inline app/src/main/AndroidManifest.xml:1]: same here

File: app/src/main/java/nethical/digipaws/Constants.java [hunk 3]
```diff
@@ -1,22 +1,22 @@
 package nethical.digipaws;
```
Comment (aikrq) [inline app/src/main/java/nethical/digipaws/Constants.java:1]: why are you using 2 space tab instead of 4?
you are already using 4 space tab in some files.

**Review by aikrq — CHANGES_REQUESTED** [review]

File: .github/workflows/runer.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: Android CI
```
Comment (aikrq) [inline .github/workflows/runer.yml:1]: Why are you adding unwanted changes?
Pull request title: **Format code**

File: app/build.gradle [hunk 5]
```diff
@@ -6,7 +6,7 @@ plugins {
 
 android {
     namespace 'nethical.digipaws'
-    compileSdk 33
+    compileSdk 34
```
Comment (aikrq) [inline app/build.gradle:9]: Same here

File: app/build.gradle [hunk 6]
```diff
@@ -28,8 +28,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```
Comment (aikrq) [inline app/build.gradle:32]: Same here

**Review by aikrq — CHANGES_REQUESTED** [review]
Please do not make unwanted changes in next time.

**Comment by nethical6** [comment]
💀💀

**Comment by nethical6** [comment]
I have reset all changes. Please fix and reopen another pr

**Pull request merged**